src/components/FactCard.tsx

```diff
@@ -0,0 +1,20 @@
+import { Section } from "../data/factData";
+import "./FactCard.scss";
+
+const FactCard = (props: { section: Section }) => {
```

Comment:
Prefer defining the props up above, like this:
```suggestion
interface FactCardProps {
  section: Section;
}

const FactCard = ({ section }: FactCardProps) => {
```

Reply:
prop interface standard followed